run-tests.pl

```diff
@@ -492,6 +488,10 @@ sub test
 {
    my ( $name, %params ) = @_;
 
+   # We expect this test to fail if it's declared to be dependent on a bug that
```

Comment:
Since `multi_test` is a separate thing, don't forget to copy this also to there.